website/en/docs/_install/compiler-babel.md

```diff
@@ -2,19 +2,19 @@
 support for Flow. Babel will take your Flow code and strip out any type
 annotations.
 
-First install `babel-cli` and `babel-preset-flow` with either
+First install `@babel/cli` and `@babel/preset-flow` with either
 [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/).
 
 ```sh
-{{include.install_command}} babel-cli babel-preset-flow
+{{include.install_command}} @babel/cli @babel/preset-flow
 ```
 
 Next you need to create a `.babelrc` file at the root of your project with
 `"flow"` in your `"presets"`.
 
 ```json
 {
-  "presets": ["flow"]
+  "presets": ["@babel/flow"]
```

Comment:
I think this needs to be `@babel/preset-flow' ?

Reply:
According to <https://babeljs.io/docs/en/babel-preset-flow>, I would agree!